### PR TITLE
Bug/s3 fix beans

### PR DESCRIPTION
**Pull request by simonhir**
**Description**

Fix s3 integration beans.

### CODE REVIEW

File: refarch-integrations/refarch-s3-integration/refarch-s3-integration-rest/refarch-s3-integration-rest-client/src/main/java/de/muenchen/refarch/integration/s3/client/repository/DocumentStorageFileRestRepository.java [hunk 1]
```diff
@@ -7,13 +7,11 @@
 import de.muenchen.refarch.integration.s3.client.repository.presignedurl.PresignedUrlRepository;
 import de.muenchen.refarch.integration.s3.client.repository.transfer.S3FileTransferRepository;
 import lombok.extern.slf4j.Slf4j;
-import org.springframework.stereotype.Repository;
 import org.springframework.web.client.HttpClientErrorException;
 import org.springframework.web.client.HttpServerErrorException;
 import org.springframework.web.client.RestClientException;
 
 @Slf4j
-@Repository
```
Comment (devtobi) [inline refarch-integrations/refarch-s3-integration/refarch-s3-integration-rest/refarch-s3-integration-rest-client/src/main/java/de/muenchen/refarch/integration/s3/client/repository/DocumentStorageFileRestRepository.java:16]: Why was this annotation removed?

File: refarch-integrations/refarch-s3-integration/refarch-s3-integration-rest/refarch-s3-integration-rest-service/src/main/java/de/muenchen/refarch/integration/s3/configuration/SecurityConfiguration.java [hunk 2]
```diff
@@ -50,6 +51,7 @@ public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
                         .permitAll())
                 .authorizeHttpRequests((requests) -> requests.requestMatchers("/**")
                         .authenticated())
+                .csrf(AbstractHttpConfigurer::disable)
```
Comment (devtobi) [inline refarch-integrations/refarch-s3-integration/refarch-s3-integration-rest/refarch-s3-integration-rest-service/src/main/java/de/muenchen/refarch/integration/s3/configuration/SecurityConfiguration.java:54]: Why must CSRF be disabled here? Isn't this risky?

**Review by devtobi — CHANGES_REQUESTED** [review]

File: refarch-integrations/refarch-s3-integration/refarch-s3-integration-rest/refarch-s3-integration-rest-service/src/main/java/de/muenchen/refarch/integration/s3/configuration/SecurityConfiguration.java (hunk 2 above)
Comment (simonhir) [reply refarch-integrations/refarch-s3-integration/refarch-s3-integration-rest/refarch-s3-integration-rest-service/src/main/java/de/muenchen/refarch/integration/s3/configuration/SecurityConfiguration.java:54]: JWT doesn'T need CSRF and if not disabled services can't communicate with each other.
See also https://github.com/it-at-m/digiwf-core/blob/dev/digiwf-libs/digiwf-spring-security/digiwf-spring-security-core/src/main/java/de/muenchen/oss/digiwf/spring/security/SecurityConfiguration.java#L79 for DigiWF

Comment (devtobi) [reply refarch-integrations/refarch-s3-integration/refarch-s3-integration-rest/refarch-s3-integration-rest-service/src/main/java/de/muenchen/refarch/integration/s3/configuration/SecurityConfiguration.java:54]: Ok thanks for clarification. ;)

File: refarch-integrations/refarch-s3-integration/refarch-s3-integration-rest/refarch-s3-integration-rest-client/src/main/java/de/muenchen/refarch/integration/s3/client/repository/DocumentStorageFileRestRepository.java (hunk 1 above)
Comment (devtobi) [reply refarch-integrations/refarch-s3-integration/refarch-s3-integration-rest/refarch-s3-integration-rest-client/src/main/java/de/muenchen/refarch/integration/s3/client/repository/DocumentStorageFileRestRepository.java:16]: Okay fine for me. :)

**Review by devtobi — DISMISSED** [review]
LGTM

File: refarch-integrations/refarch-s3-integration/refarch-s3-integration-rest/refarch-s3-integration-rest-service/src/main/java/de/muenchen/refarch/integration/s3/configuration/SecurityConfiguration.java (hunk 2 above)
Comment (simonhir) [reply refarch-integrations/refarch-s3-integration/refarch-s3-integration-rest/refarch-s3-integration-rest-service/src/main/java/de/muenchen/refarch/integration/s3/configuration/SecurityConfiguration.java:54]: After looking a bit around, this should be automatically set if providing Bearer token.
I'll try it again and probably remove it. Was maybe my fault.

Comment (simonhir) [reply refarch-integrations/refarch-s3-integration/refarch-s3-integration-rest/refarch-s3-integration-rest-service/src/main/java/de/muenchen/refarch/integration/s3/configuration/SecurityConfiguration.java]: Reverted the CSRF disable.

**Review by devtobi — APPROVED** [review]
LGTM

**Pull request merged**